tests/typings/cucumber/config.ts

```diff
@@ -3,9 +3,7 @@ import { expectType } from 'tsd'
 const config: WebdriverIO.Config = {
     cucumberOpts: {
         timeout: 123,
-        require: ['123'],
-        // @ts-expect-error
-        scenarioLevelReporter: 'wrong param'
```

Comment:
Why removing this test?

Reply:
Oops, my mistake. While piecing things together to restore the old features, I accidentally missed to add it back.